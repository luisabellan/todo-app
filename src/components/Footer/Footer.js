import React from 'react';

function Footer() {
  return (
    <div data-testid="footer" >
      <footer className="footer" >
        <span className="copyright">
          Made with ðŸ’“ and â˜• by{' '}
          <a href="https://luisabellan.com">Luis Abellan</a>
        </span>
      </footer>
    </div>

  );
}

export default Footer;
