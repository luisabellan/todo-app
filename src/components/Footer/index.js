import React from 'react';

function Footer() {
  return (
    <div>
      <footer className="footer" data-testid="footer" >
        <span className="copyright">
          Made with ðŸ’“ and â˜• by{' '}
          <a href="https://luisabellan.com">Luis Abellan</a>
        </span>
      </footer>
    </div>

  );
}

export default Footer;
