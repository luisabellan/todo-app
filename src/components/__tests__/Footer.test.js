import React from 'react';
import '@testing-library/jest-dom';
import userEvent from '@testing-library/user-event';
import { render, screen, cleanup } from '@testing-library/react';
import Footer from '../Footer/Footer';

const todo = {
  id: 1,
  name: 'buy bread',
  completed: false
};

afterEach(cleanup);

describe('todo test', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('should show footer', () => {
    render(<Footer />);

    expect(screen.getByTestId("footer")).toBeInTheDocument();
  });
});
