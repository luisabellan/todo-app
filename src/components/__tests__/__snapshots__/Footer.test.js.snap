// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`todo test matches snapshot 1`] = `
<DocumentFragment>
  <div
    data-testid="footer"
  >
    <footer
      class="footer"
    >
      <span
        class="copyright"
      >
        Made with ðŸ’“ and â˜• by 
        <a
          href="https://luisabellan.com"
        >
          Luis Abellan
        </a>
      </span>
    </footer>
  </div>
</DocumentFragment>
`;
